test/interpreter/function-varpa.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VARPA', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARPA()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate variance (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARPA(2, 3)'],
+      ['=VARPA(1)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.25)
+    expect(engine.getCellValue(adr('A2'))).toEqual(0)
+  })
+
+  it('should coerce explicit argument to numbers', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARPA(2, 3, 4, TRUE(), FALSE(), "1",)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(1.95918367346939, 6)
+  })
+
+  it('should ignore non-numeric values in ranges', () => {
```

Comment:
VAR.P has the same description and the same values but different result. I think that the description is not precise enough. One includes logical values the other dosn't.

From "If you do not want to include logical values and text representations of numbers in a reference as part of the calculation, use the VARP function." I would say that here w should mention that this INCLUDES logical values and text representation

```suggestion
  it('should evaluate TRUE to 1, FALSE to 0 and coerce text representation of numbers', () => {
```